share/autostart

```diff
@@ -121,7 +121,10 @@ hc set frame_transparent_width 5
 hc set frame_gap 4
 
 hc attr theme.title_height 15
-hc attr theme.title_font '-*-fixed-medium-r-*-*-13-*-*-*-*-*-*-*'
+hc attr theme.title_font 'Dejavu Sans:pixelsize=12'  # example using Xft
+# hc attr theme.title_font "Liberation Mono:pixelsize=12:antialias=false:autohint=false"
+# hc attr theme.title_font '-*-fixed-medium-r-*-*-13-*-*-*-*-*-*-*'
+hc attr theme.padding_top 2
```

Comment:
Why is this added?

Reply:
I've restricted the font examples to two (one xft, one XLFD). The `text_height` only is the height of the text above it's baseline, and the `padding_top` adds space above the window content (but still below the title). So `padding_top` is used as `text_depth` here.

Reply:
I see, thanks!